tests/.deprc

```diff
@@ -0,0 +1,4 @@
+  {
+    "root": "tests",
```

Comment:
yes this works

On Wed, Mar 11, 2015 at 7:46 PM, Joel Kemp <notifications@github.com> wrote:

> In tests/.deprc
> <https://github.com/mrjoelkemp/Dependents/pull/131#discussion_r26274763>:
>
> > @@ -0,0 +1,4 @@
> > +  {
> > +    "root": "tests",
>
> Assuming you could also just do ./ if you're opening this up in its own
> directory
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mrjoelkemp/Dependents/pull/131/files#r26274763>.
>



-- 
Cheers, Cuidate, TTFN <http://www.youtube.com/watch?v=5Gu50vq5ux4>
David